.github/workflows/e2e-aws-management-and-workload-cluster.yaml

```diff
@@ -1,6 +1,7 @@
 name: E2E Test - AWS Management and Workload Cluster
 
 on:
+  workflow_dispatch:
```

Comment:
I think you are missing something here like and event?

Reply:
Maybe some additional context, the github action to trigger this event in the daily build uses the `repository_dispatch`.

Reply:
Ah okay, I missed to add details about this in the PR description, my bad. I added this to enable manual trigger of the pipeline . The manual trigger can help anyone trigger from web or CLI manually, so it's not based on any other event like `daily-build`, but one can introduce inputs for manual triggers but in this case I don't see any inputs. More info - https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch , https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow

Reply:
I wanted to add this manual trigger feature for all pipelines, so that one can trigger it without other events (like push, etc)

But in some workflows, like vSphere one, there's a `repository_dispatch` where we can trigger by simulating / emitting that event using the GitHub API without the official daily build triggering it with the https://github.com/peter-evans/repository-dispatch action

Reply:
Let me know if that makes sense

Reply:
It will look something like this for example (my fork where I have complete access) - 

![Screenshot 2022-01-27 at 5 08 08 PM](https://user-images.githubusercontent.com/12808424/151351679-2c38576b-f739-4083-8bb8-6e4054b24076.png)

Reply:
ok gotcha makes sense!